rdmo/management/utils.py

```diff
@@ -0,0 +1,9 @@
+
+def replace_uri_in_template_string(template: str, source_uri: str, target_uri: str) -> str:
+    patterns = [f"'{source_uri}'", f'"{source_uri}"']
```

Comment:
You could also do

```
replacements = [
    (f"'{source_uri}'", f"'{target_uri}'"),
    (f'"{source_uri}"', f'"{target_uri}"'),
]

for pattern, replacement in replacements:
    ...
```

(or keep the current one, I don't mind)

Reply:
yes, looks more readable